examples/rosettacode/bitmap.nit

```diff
@@ -0,0 +1,247 @@
+# The Bitmap class represents a 24-bit bitmap image. An instance can be constructed
+# by calling load or with_size:
+#
+# 	new Bitmap.load(path_to_a_bmp_file)
+#   new Bitmap.with_size(400, 300)
+#
+# The width and height attributes contain the image's width and height,
+# respectively.
+#
+# Individual pixels can be manipulated by calling the set_pixel function:
+#
+#	set_pixel(x, y, color)
+#
+# The no-argument grayscale function converts the bitmap to grayscale and is an
+# implementation of this Rossetacode task:
+# http://rosettacode.org/wiki/Grayscale_image
+#
+# Finally, the bitmap can be saved to a file by calling save:
+#
+#	save(path_to_a_file)
+#
+# For information on the bitmap format, see
+# 	http://en.wikipedia.org/wiki/BMP_file_format
+module bitmap
+
+class Bitmap
```

Comment:
not doc for the class. Maybe part of the for of the module should be here instead.
Or at least add a short doc